host-bmc/dbus_to_host_effecters.cpp

```diff
@@ -287,7 +287,7 @@ int HostEffecterParser::sendSetStateEffecterStates(
                 std::cerr << "Failed to decode setStateEffecterStates response,"
                           << " rc " << rc << "\n";
                 pldm::utils::reportError(
-                    "xyz.openbmc_project.bmc.pldm.SetHostEffecterFailed",
+                    "xyz.openbmc_project.Pldm.Error.SetHostEffecterFailed",
```

Comment:
Can this be PLDM.Error ? all in capitals?
Same comment in every place

Reply:
Done changed in all files